packages/react/src/Switch/README.md

```diff
@@ -0,0 +1,5 @@
+# React Switch Component
+
+The React `<Switch />` component is a wrapper around the `<input type="checkbox" />`. It can be used as a standalone component or as a form control element. When used as a standalone component provide a `aria-labeledby` prop to keep it accessible. When used as a formcontrol, make sure to add the associated `<Label>` component.
```

Comment:
```suggestion
The React `<Switch />` component is a wrapper around the `<input type="checkbox" />`. It can be used as a standalone component or as a form control element. When used as a standalone component provide an `aria-label` or `aria-labelledby` prop to keep it accessible. When used as a formcontrol, make sure to add the associated `<FormLabel>` component.
```